com.unity.visualeffectgraph/Editor/Inspector/VFXAssetEditor.cs

```diff
@@ -421,6 +421,96 @@ private string UpdateModeToString(VFXUpdateMode mode)
 
     private static readonly float k_MinimalCommonDeltaTime = 1.0f / 800.0f;
 
+    public static void PrewarmGenericInspector(SerializedObject resourceObject, SerializedProperty prewarmDeltaTime, SerializedProperty prewarmStepCount)
```

Comment:
Not sure about the naming here. Wouldnt Display/UpdatePrewarmGUI be more explicit?

Reply:
Fix at c42c384f33a0718ffd4a67f69be19be12d8a8083